configuration/components/loki.libsonnet

```diff
@@ -294,6 +301,7 @@ local defaults = {
       per_stream_rate_limit: '3MB',
       per_stream_rate_limit_burst: '15MB',
       split_queries_by_interval: '30m',
+      deletion_mode: 'disabled',
     },
     querier: {
```

Comment:
According to upgrade docs (https://grafana.com/docs/loki/latest/upgrading/#engine-query-timeout-is-deprecated) we should drop `query.engine.timeout` in favor of `query.query_timeout`. This is probably a hindsight from the last update to 2.7.0

Reply:
I saw that. Thing is, I don't know which value to keep. WDYT the value should be?

Reply:
we should keep the `query.query_timeout` with the value as is.